core/src/HIP/Kokkos_HIP_Instance.cpp

```diff
@@ -101,30 +101,29 @@ int HIPInternal::was_initialized = 0;
 int HIPInternal::was_finalized   = 0;
 //----------------------------------------------------------------------------
 
-void HIPInternal::print_configuration(std::ostream & /*s*/) const {
-  // FIXME_HIP
-  Kokkos::abort("print_configuration not implemented!\n");
-  /*const HIPInternalDevices & dev_info = HIPInternalDevices::singleton();
+void HIPInternal::print_configuration(std::ostream &s) const {
+  const HIPInternalDevices &dev_info = HIPInternalDevices::singleton();
 
-#if defined( KOKKOS_ENABLE_HIP )
-    s << "macro  KOKKOS_ENABLE_HIP      : defined" << std::endl ;
+#if defined(KOKKOS_ENABLE_HIP)
```

Comment:
It feels superfluous to check that the macro is defined.  Wouldn't be there otherwise.